README.md

```diff
@@ -147,7 +147,15 @@ credentials.json
           "brokerCertificate":"",
           "clientCertificate":""
         }
-   }
+   },
+  "QE MQTT": {
```

Comment:
Pls check how credentials are added, when AMQ broker is deployed and add the mqtt config accordingly.